README.md

```diff
@@ -1 +1,37 @@
-# dissolution-web
\ No newline at end of file
+# Node Skeleton Frontend
+This repo contains a skeleton Companies House NodeJS frontend project.
+
+## Technologies
+
+- [NodeJS](https://nodejs.org/)
+- [ExpressJS](https://expressjs.com/)
+- [NunJucks](https://mozilla.github.io/nunjucks)
+- [GulpJS](https://gulpjs.com/)
+- [Inversify](https://github.com/inversify/)
+
+## Recommendations
+
+We recommend the use of Visual Studio Code for development as it allows the installation of the TSLint and the Nunjucks plugins. These plugins will make linting of TS and Nunjuck much better than mmost code editors.
+
+IntelliJ does not have a Nunjuck plugin but you can configure IntelliJ to provide syntax highlighting using Twig plugin
+
+## How to run it
```

Comment:
Just a note for anyone else looking at this, this is temporary until the Docker env is in place